File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U30EmrTagSaveLayoutInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
    public class OCS2015U30EmrTagSaveLayoutInfo
    {
        private String _tagId;
        private String _tagName;
        private String _tagCode;
        private String _tagDisplayText;
        private String _description;
        private String _tagLevel;
        private String _tagParent;
        private String _filterFlg;
        private String _rowState;
        private String _sysId;

        public String TagId
        {
            get { return this._tagId; }
            set { this._tagId = value; }
        }

        public String TagName
        {
            get { return this._tagName; }
            set { this._tagName = value; }
        }

        public String TagCode
        {
            get { return this._tagCode; }
            set { this._tagCode = value; }
        }

        public String TagDisplayText
        {
            get { return this._tagDisplayText; }
            set { this._tagDisplayText = value; }
        }

        public String Description
        {
            get { return this._description; }
            set { this._description = value; }
        }

        public String TagLevel
        {
            get { return this._tagLevel; }
            set { this._tagLevel = value; }
        }

        public String TagParent
        {
            get { return this._tagParent; }
            set { this._tagParent = value; }
        }

        public String FilterFlg
        {
            get { return this._filterFlg; }
            set { this._filterFlg = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public OCS2015U30EmrTagSaveLayoutInfo() { }

        public OCS2015U30EmrTagSaveLayoutInfo(String tagId, String tagName, String tagCode, String tagDisplayText, String description, String tagLevel, String tagParent, String filterFlg, String rowState, String sysId)
        {
            this._tagId = tagId;
            this._tagName = tagName;
            this._tagCode = tagCode;
            this._tagDisplayText = tagDisplayText;
            this._description = description;
            this._tagLevel = tagLevel;
            this._tagParent = tagParent;
            this._filterFlg = filterFlg;
            this._rowState = rowState;
            this._sysId = sysId;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U30EmrTagSaveLayoutInfo.java
package nta.med.data.model.ihis.emr;

import java.math.BigDecimal;

public class OCS2015U30EmrTagSaveLayoutInfo {
	private Integer tagId;
	private String tagName;
	private String tagCode;
	private String tagDisplayText;
	private String description;
	private String tagLevel;
	private String tagParent;
	private BigDecimal filterFlg;
	private String rowState;
	private String sysId;
	public OCS2015U30EmrTagSaveLayoutInfo(Integer tagId, String tagName,
			String tagCode, String tagDisplayText, String description,
			String tagLevel, String tagParent, BigDecimal filterFlg,
			String rowState, String sysId) {
		super();
		this.tagId = tagId;
		this.tagName = tagName;
		this.tagCode = tagCode;
		this.tagDisplayText = tagDisplayText;
		this.description = description;
		this.tagLevel = tagLevel;
		this.tagParent = tagParent;
		this.filterFlg = filterFlg;
		this.rowState = rowState;
		this.sysId = sysId;
	}
	public Integer getTagId() {
		return tagId;
	}
	public void setTagId(Integer tagId) {
		this.tagId = tagId;
	}
	public String getTagName() {
		return tagName;
	}
	public void setTagName(String tagName) {
		this.tagName = tagName;
	}
	public String getTagCode() {
		return tagCode;
	}
	public void setTagCode(String tagCode) {
		this.tagCode = tagCode;
	}
	public String getTagDisplayText() {
		return tagDisplayText;
	}
	public void setTagDisplayText(String tagDisplayText) {
		this.tagDisplayText = tagDisplayText;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getTagLevel() {
		return tagLevel;
	}
	public void setTagLevel(String tagLevel) {
		this.tagLevel = tagLevel;
	}
	public String getTagParent() {
		return tagParent;
	}
	public void setTagParent(String tagParent) {
		this.tagParent = tagParent;
	}
	public BigDecimal getFilterFlg() {
		return filterFlg;
	}
	public void setFilterFlg(BigDecimal filterFlg) {
		this.filterFlg = filterFlg;
	}
	public String getRowState() {
		return rowState;
	}
	public void setRowState(String rowState) {
		this.rowState = rowState;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	
}
